### DISCUSSION TITLE
Content Library

### DISCUSSION THREAD

**Discussion by bweissman** [dataplat/DataSaturdays#161] [Ideas]
I would love to see a library that brings all the content from events together

Ideally with the option to either upload or just link content (on github, personal blogs, wherever).

When google searching for presentations, there is a lot of clutter - this could be highly focussed. 

Thoughts?

**Comment by SQLDBAWithABeard** [comment]
Awesome suggestion - How do we enable this

**Comment by gavincampbell** [comment]
1) Invite speakers to link content on the event page via PR
2) Do magic to collate all these links onto datasaturday.com/library
3) ....

[a comment by an automated account is not shown]
